templates/content/news/2021-10-29-egm-results.md

```diff
@@ -0,0 +1,42 @@
+---
+title: October 2021 EGM Results
+---
+
+On Friday 29 October 2021 we held an Extraordinary General Meeting (EGM) to elect a new Press & Publicity Officer, one Ordinary Member, as well as pass an amendment to the constitution.
+
+Candidates met on Google Meet to present their manifesti, while the meeting was streamed using our in-house RTMP service with a latency of around 10 seconds, and our Lightspeed service with less than a second of delay. Questions from members were taken via Slack. Frequent pauses were given to allow any livestream delay to catch up before moving on to the next item.
+
+## Amendments
+
+One amendment was presented to the society and accepted. The passing of an amendment required the support of two-thirds of the members voting. You can learn more about the amendment in the linked pull request.
+
+ - [Code of Conduct](https://github.com/HackSoc/constitution/pull/16)
+   - **Final vote:** 12 for (100%), 0 against, 0 abstaining
+   - **Passed**
+
+## Committee Election Results
+
+All positions were voted on using the Single Transferable Vote (STV) method, which is equivalent to the Alternative Vote (AV) method for positions with one seat (which in this case was all positions, since only one Ordinary Member seat was up for election). All positions had the option to Re-Open Nominations (RON). Quorum was **10** based on our membership count at the time, which was achieved for all elections.
+
+Voters were given access to a Google Form for each election, which required them to be signed into their University account, and allowed paid members to cast at most one vote per election. The elections were open for ten minutes at a time, then closed to prevent further submissions. The results of the Google Forms were collected into a spreadsheet, which was then used to count the votes manually, or electronically in the case of the Ordinary Member election. Voters' email addresses were compared against a list of HackSoc members, to ensure that all voters were eligible to vote. All votes will later be verified manually. 
```

Comment:
> a spreadsheet, which was then used to count the votes manually, or electronically in the case of the Ordinary Member election

is this still the case this year?

Reply:
well, I looked at some pie charts to determine the result, so technically yes ;^)

(valid point, will change, how about:)

```suggestion
Voters were given access to a Google Form for each election, which required them to be signed into their University account, and allowed paid members to cast at most one vote per election. The elections were open for ten minutes at a time, then closed to prevent further submissions. The results of the Google Forms were collected into a spreadsheet, which was then used to count the votes manually. Voters' email addresses were compared against a list of HackSoc members, to ensure that all voters were eligible to vote. All votes will later be verified manually. 
```

Reply:
sounds good